livesql/binlog.go

```diff
@@ -103,6 +103,7 @@ func NewBinlogWithSource(ldb *LiveDB, sourceDB *sql.DB, host string, port uint16
 	syncer := replication.NewBinlogSyncer(&replication.BinlogSyncerConfig{
 		ServerID: binary.LittleEndian.Uint32(slaveId),
 		Host:     host,
+		Localhost: "localhost",
```

Comment:
I'm wondering if this could have any adverse effects if this were to be used in an AWS/remote binlog context. Should we just truncate the result of `os.Hostname` to be below the size limit?

Reply:
sure, updated to use a substring of `os.Hostname`